.vscode/settings.json

```diff
@@ -6,6 +6,15 @@
         "unordered_map": "cpp",
         "vector": "cpp",
         "string_view": "cpp",
-        "initializer_list": "cpp"
+        "initializer_list": "cpp",
```

Comment:
don't add these files when they pop up; should only commit files you directly change, not config files